src/extension/background-script/actions/nostr/getPublicKey.ts

```diff
@@ -0,0 +1,8 @@
+import state from "~/extension/background-script/state";
+
+const getPublicKey = async () => {
+  const publicKey = state.getState().getNostr().getPublicKey();
+  return publicKey;
```

Comment:
don't we need to always return an object with a `data` (or `error`) key?

Reply:
Good point. I think this is necessary for actions that are [called/used by utils/call](https://github.com/reneaaron/lightning-browser-extension/blob/feature/nostr/src/common/lib/utils.ts#L34). This is expecting a `data` prop.  
In this case it's only just called directly by `getPublicKeyOrPrompt`. So if `getPublicKey` is never called via a `message` does it need it own action-file? Would it be ok to move it into `getPublicKeyOrPrompt`?

Reply:
Usually yes, but this function was only used internally. Changed usages to use state.getPublicKey() directly and removed this function completely.